code/modules/client/preferences_savefile.dm

```diff
@@ -51,6 +51,8 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		toggles |= SOUND_ALT
 	if (current_version < 37)
 		chat_toggles |= CHAT_TYPING_INDICATOR
+	if (current_version < 38)
+		chat_toggles |= HEAR_TTS // Muted by default
```

Comment:
wouldn't this turn on HEAR_TTS

Reply:
from DM reference:
usr.sight |= BLIND // turn on the blind bit

turns on if not on, if already on does nothing